packages/virtual-core/src/index.ts

```diff
@@ -328,6 +329,7 @@ export class Virtualizer<
   options!: Required<VirtualizerOptions<TScrollElement, TItemElement>>
   scrollElement: TScrollElement | null = null
   targetWindow: (Window & typeof globalThis) | null = null
+  private enabled: boolean = true
```

Comment:
I think we don't need an instance variable, overall we should relay on scrollElement if it's null then everything should be reseted.

enabled={false} should be same as getScrollElement: () => null

Reply:
Yes, I guess that's a correct flow